packages/payload/src/fields/config/types.ts

```diff
@@ -1520,6 +1532,7 @@ export type BlocksField = {
   blockReferences?: (Block | BlockSlug)[]
   blocks: Block[]
   defaultValue?: DefaultValue
+  filterOptions?: BlocksFilterOptions
```

Comment:
Reason for keeping name `filterOptions` despite different type is in PR description